lib/lokka/models/entry.rb

```diff
@@ -1,55 +1,65 @@
-class Entry < ActiveRecord::Base
-  attr_accessible :user,
-                  :category_id,
-                  :slug,
-                  :title,
-                  :body,
-                  :markup,
-                  :type,
-                  :draft,
-                  :created_at,
-                  :updated_at
+# frozen_string_literal: true
 
+class Entry < ActiveRecord::Base
   has_many :comments
-  has_many :tags, through: :taggings
   has_many :taggings, as: :taggable
+  has_many :tags, through: :taggings
 
   belongs_to :user
   belongs_to :category
 
-  validates :title, presence:   true
+  validates :title, presence: true
   validates :slug, uniqueness: true,
-                   format: %r{^[_/0-9a-zA-Z-]+$}, allow_blank: true
+                   format: %r{\A[_/0-9a-zA-Z-]+\z}, allow_blank: true
 
   validate :validate_confliction
   after_save :update_fields
 
-  default_scope { order('created_at DESC') }
-  scope :published,   ->{ where(draft: false) }
-  scope :unpublished, ->{ where(draft: true) }
-  scope :posts,       ->{ where(type: 'Post') }
-  scope :pages,       ->{ where(type: 'Page') }
+  default_scope { order(created_at: :desc) }
+  scope :published,   -> { where(draft: false) }
+  scope :unpublished, -> { where(draft: true) }
+  scope :posts,       -> { where(type: 'Post') }
+  scope :pages,       -> { where(type: 'Page') }
   scope :recent,
-    ->(count = 5){ limit(count) }
+        ->(count = 5) { limit(count) }
   scope :between_a_year,
-    ->(time){
-    where(created_at: time.beginning_of_year..time.end_of_year)
-  }
+        ->(time) {
+          where(created_at: time.beginning_of_year..time.end_of_year)
+        }
   scope :between_a_month,
-    ->(time){
-    where(created_at: time.beginning_of_month..time.end_of_month)
-  }
+        ->(time) {
+          where(created_at: time.beginning_of_month..time.end_of_month)
+        }
   scope :search,
-    ->(word){
-      where('title LIKE ?', "#{word}") | where('body LIKE ?', "#{word}")
-  }
+        ->(word) {
+          where('title LIKE ?', "%#{word}%").or(where('body LIKE ?', "%#{word}%"))
+        }
 
   def self.get_by_fuzzy_slug(id_or_slug)
     where(slug: id_or_slug).first || where(id: id_or_slug).first
   end
 
+  def raw_body
```

Comment:
This is for reviving `apply_continue_reading`. This method prevent from `StackLevelTooDeep`